com.hazelcast.jet.impl.processor.SessionWindowPTest.when_disorderedEVentsWithThreeKeys

```
Error Message
call to finishSnapshotRestore() took 1045.7ms, it should be <1000ms
Stacktrace
java.lang.AssertionError: call to finishSnapshotRestore() took 1045.7ms, it should be <1000ms
	at com.hazelcast.jet.impl.processor.SessionWindowPTest.assertCorrectness(SessionWindowPTest.java:153)
	at com.hazelcast.jet.impl.processor.SessionWindowPTest.when_disorderedEVentsWithThreeKeys(SessionWindowPTest.java:107)
```

https://hazelcast-l337.ci.cloudbees.com/view/Jet/job/jet-pr-builder/com.hazelcast.jet$hazelcast-jet-core/3017/testReport/junit/com.hazelcast.jet.impl.processor/SessionWindowPTest/when_disorderedEVentsWithThreeKeys/

--- a/hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/test/TestSupport.java
+++ b/hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/test/TestSupport.java
@@ -177,9 +177,9 @@ public final class TestSupport {
     private static final Address LOCAL_ADDRESS;
 
     // 1ms should be enough for a cooperative call. We warn, when it's more than 5ms and
-    // fail when it's more than 100ms, possibly due to other  activity in the system, such as
-    // parallel tests or GC.
-    private static final long COOPERATIVE_TIME_LIMIT_MS_FAIL = 1000;
+    // fail when it's more than 100ms, possibly due to other activity in the system, such as
+    // tests running in parallel or a GC.
+    private static final long COOPERATIVE_TIME_LIMIT_MS_FAIL = 5_000;
     private static final long COOPERATIVE_TIME_LIMIT_MS_WARN = 5;
 
     private static final long BLOCKING_TIME_LIMIT_MS_WARN = 10000;

--- a/hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/MasterContext.java
+++ b/hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/MasterContext.java
@@ -913,7 +913,8 @@ public class MasterContext {
         // If there is any member in our participants that is not among current data members,
         // this job will be restarted anyway. If it's the other way, then the sizes won't match.
         if (executionPlanMap == null || executionPlanMap.size() == currentDataMembers.size()) {
-            LoggingUtil.logFine(logger, "Not up-scaling job %s: already running on all members", jobIdString());
+            LoggingUtil.logFine(logger, "Not up-scaling job %s: not running or already running on all members",
+                    jobIdString());
             return false;
         }
 

